addons/kotsadm/0.9.4/tmpl-secret-password.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kotsadm-password
+stringData:
+  passwordBcrypt: ${BCRYPT_PASSWORD}
```

Comment:
Am I reading this right that this is a bcrypted password?   If it is, how is it used?

Reply:
The first time the kotsadm addon is installed a password is generated and then bcrypted. The bcrypted password is used as an environment variable on the kotsadm-api pod and the user has to enter the plain password in the UI the first time.

Reply:
https://github.com/replicatedhq/kots/blob/master/pkg/kotsadm/api_objects.go#L128